src/na/zoned/decomposer/NativeGateDecomposer.cpp

```
@@ -0,0 +1,276 @@
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * Licensed under the MIT License
+ */
+
+//
+// Created by cpsch on 11.12.2025.
+//
+#include "na/zoned/decomposer/NativeGateDecomposer.hpp"
+
+// #include "ir/operations/StandardOperation.hpp"
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)